static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -1022,17 +1042,24 @@ const Body = styled(Layout.Body)`
 `;
 
 const Main = styled(Layout.Main)`
+  max-width: 1000px;
   padding: ${space(4)};
+  flex: 1;
```

Comment:
do we need `flex:1`?

Reply:
For now, yes because `grid-template-rows: 1fr` lets the main and side components reach all the way to the bottom, and `flex: 1` also contributes to making sure the form makes use of that space and positions the footer down as well within that grid row

I can follow up on it because mixing grid and flexbox in this way does get a little confusing